client/packages/system/src/Name/ListView/Facilities/AppBarButtons.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import {
+  AppBarButtonsPortal,
+  Grid,
+  ButtonWithIcon,
+  ToggleState,
+  UploadIcon,
+  useTranslation,
+  NamePropertyNode,
+  useNotification,
+} from '@openmsupply-client/common';
+
+export const AppBarButtonsComponent = ({
+  importModalController,
+  properties,
+  propertiesLoading,
+}: {
+  importModalController: ToggleState;
+  properties: NamePropertyNode[] | undefined;
+  propertiesLoading: boolean;
+}) => {
+  const t = useTranslation();
+  const { error } = useNotification();
+
+  const handleClick = () => {
+    properties?.length && properties.length >= 0
+      ? importModalController.toggleOn()
+      : error(t('error.no-properties-to-import'))();
```

Comment:
minor lol but could it be info() rather than error(), so its the blue snack? no properties doesn't mean anything has gone wrong, they just aren't enabled...